harvard-cranfield-university.csl

```diff
@@ -270,25 +257,35 @@
             </group>
             <text variable="archive"/>
             <text variable="archive_location"/>
-            <text macro="online"/>
             <text macro="publisher"/>
           </group>
         </else-if>
-        <else-if type="article-journal article-magazine article-newspaper broadcast interview manuscript map patent personal_communication song speech thesis webpage motion_picture" match="any">
-          <group suffix=".">
+        <else-if type="article-journal" match="any">
+          <group delimiter=", ">
+            <text macro="title" prefix="'" suffix="'"/>
```

Comment:
I assume these are single quotes? Please set them with quotes="true" -- both to get proper smart quotes and for proper handling in the processor.